Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
Looking at [some assertion libraries supported by mocha](http://mochajs.org/#assertions), it seems to me as if [chai](http://chaijs.com/) could model the current assertion syntax most closely. So we'd have to replace the matcher implementation, but could leave the tests in place. On the other hand, [unexpected](http://unexpected.js.org/) sounds like it could express our semantics quite well, and in an even more readable and flexible fashion. I'm leaning slightly towards the latter. But in any case I'd like to see this PR here landed first.